index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body class="container">
+    <header class="header">
+        <p class="name">Fernando Herrera</p>
+        <div>
+            <div class="hamb hamb1"></div>
+            <div class="hamb hamb2"></div>
+            <div class="hamb hamb3"></div>            
+        </div>     
+    </header>
+    <main >
```

Comment:
- [ ] Thank you for implementing the headline section 👍. however, please use a `section` here instead of a `div`.
something like this would be nice. 
```suggestion
    <section >
```

Reply:
Hi, :+1:  I believe that I have to change a div tag and not the main tag. :-)